stacker/config/__init__.py

```diff
@@ -103,6 +113,24 @@ def parse(raw_config):
 
     """
 
+    # Convert any applicable dictionaries back into lists
+    config_dict = yaml.safe_load(raw_config)
+    if config_dict.get('stacks') and isinstance(config_dict['stacks'], dict):
+        stack_list = []
+        for key, value in config_dict['stacks'].iteritems():
```

Comment:
I feel like this is repeated code below - probably worth turning into a method that can be shared.